src/pool/spawn.rs

```diff
@@ -120,6 +129,68 @@ impl<T> QueueCore<T> {
             }
         }
     }
+
+    /// Scale workers.
+    pub fn scale_workers(&self, new_thread_count: usize) -> bool {
+        let _ = self.l.lock().unwrap();
+        if new_thread_count < self.config.min_thread_count
+            || new_thread_count > self.config.max_thread_count
+        {
+            return false;
+        }
+
+        let current_thread_count = self
+            .config
+            .core_thread_count
+            .swap(new_thread_count, Ordering::SeqCst);
+        let gap = new_thread_count as isize - current_thread_count as isize;
+        if gap > 0 {
+            let addr = self as *const QueueCore<T> as usize | 1;
```

Comment:
Why not `+ 1`?

Reply:
I think both are okay? What are the benefits of choosing  `+1`?  I remember this value is used as a number.

Reply:
`| 1` assumes address always be even.

Reply:
Oh, you are right, it will be fixed, thanks!